README.md

```diff
@@ -111,49 +141,17 @@ wake = WakewordDetector("path_to_tflite_model")
 
 ### Automatic Speech Recognition (ASR)
 
-Automatic Speech Recognition is a technique used to turn speech into a transcript. You can initialize our cloud ASR with:
+Automatic Speech Recognition is a technique used to turn speech into a transcript. For this part you will need to create an account [here](https://www.spokestack.io/create) if you already haven't. You can initialize our cloud ASR with:
 
 ```python
 from spokestack.asr.spokestack.speech_recognizer import SpeechRecognizer
 
 asr = SpeechRecognizer("spokestack_id", "spokestack_secret")
 ```
 
-### Speech Pipeline
-
-The Speech Pipeline is the component that ties together VAD, Wakeword, and ASR. Essentially, this component listens to a frame of audio to determine if speech is present. If speech is detected, the Wakeword model processes the subsequent frames of audio looking for the specific keyword. If the keyword is found, the pipeline is activated and converts the following audio into a transcript. The Speech Pipeline is established like this from the previous initialized components:
-
-```python
-from spokestack.pipeline import SpeechPipeline
-
-pipeline = SpeechPipeline(mic, [vad, wake, asr])
-pipeline.start()
-pipeline.run()
-```
-
-#### Speech Context
-
-Speech context manages the state of the pipeline. For example, when the an activation event is triggered, the `is_active` property in the Speech Context is set to `True`. The initialization of `SpeechContext` is handled by the pipeline, but if required elsewhere, can be initialized like this:
-
-```python
-from spokestack.context import SpeechContext
-
-context = SpeechContext()
-```
-
-#### Pipeline Callbacks
-
-Pipeline callbacks allow additional code to be executed when a speech event is detected. For example, we can print when the pipeline is activated by registering a function with the `pipeline.event` decorator.
-
-```python
-@pipeline.event
-def on_activate(context):
-    print(context.is_active)
-```
-
 ### Natural Language Understanding (NLU)
 
-Natural Language Understanding turns an utterance into a machine readable format. For our purposes, this is joint intent detection and slot filling. We like to think of intents as the action a user desires from an application, and slots as the optional arguments to fulfill the requested action. Our NLU model is initialized like this:
+Natural Language Understanding turns an utterance into a machine readable format. For our purposes, this is joint intent detection and slot filling. You can read more about the concepts [here](https://www.spokestack.io/docs/concepts/nlu). We like to think of intents as the action a user desires from an application, and slots as the optional arguments to fulfill the requested action. Our NLU model is initialized like this:
```

Comment:
How does this fit in with the pipeline introduced above?

Reply:
I guess it doesn't without the Spokestack abstraction.

Reply:
Oh, I wasn't suggesting that it _doesn't_ fit, just that a snippet that uses the NLU in a pipeline transcript callback would be helpful to the reader.

The Spokestack abstraction is another layer of developer ergonomics, but the library's totally usable without it, especially in Python where initialization of the separate components doesn't take up a screen of code all by itself.

Reply:
so I guess what would be good for the readme is like an abridged tutorial. overview of all the features, but super brief.

Reply:
That's what I find most useful in a readme. As @noelweichbrodt has pointed out, this holds the potential for getting out of sync with the codebase, so sticking to the highest conceptual level possible is best, but I still think it's good to give just enough depth that the readme can provide a working, paste-able example of the library's main value proposition(s).

Link to the docs for more details, but provide enough value in the readme that the reader _wants_ to make that extra click.

Reply:
ok I think I understand. sorry this was such a big deal.

Reply:
No problem at all—I hate to keep asking for changes, but I have...strong opinions about words (and like to present as unified a front as possible across the platforms).